batch/sql/estimated-current.sql

```diff
@@ -360,10 +360,6 @@ BEGIN
 
   SET msec_mcpu_diff = msec_diff * job_cores_mcpu;
 
-  UPDATE batches
-  SET msec_mcpu = batches.msec_mcpu + msec_mcpu_diff
-  WHERE id = NEW.batch_id;
-
   UPDATE jobs
```

Comment:
Should we just delete the update from jobs as well?

Reply:
done